src/backend/distributed/planner/multi_explain.c

```diff
@@ -275,6 +275,22 @@ ExplainSubPlans(DistributedPlan *distributedPlan, ExplainState *es)
 			es->indent += 3;
 		}
 
+		if (es->analyze)
+		{
+			if (es->timing)
+			{
+				ExplainPropertyFloat("Subplan Duration", "ms", subPlan->durationMillisecs,
+									 2, es);
+			}
+
+			ExplainPropertyInteger("Intermediate Data Size", "bytes",
+								   subPlan->bytesSentPerWorker, es);
+			ExplainPropertyInteger("Receiver Node Count", NULL,
+								   subPlan->remoteWorkerCount,
```

Comment:
Perhaps we could turn this and the following into a single string property to help explain what this means.

e.g.
```
Result destination: Send to %d nodes
Result destination: Write locally
Result destination: Send to %d nodes, write locally
```

Reply:
done.